parts/kubernetesmastercustomscript.sh

```diff
@@ -169,12 +169,6 @@ function configNetworkPolicy() {
     fi
 }
 
-function ensureEtcd() {
```

Comment:
I thought the VM wasn't finished provisioning until CustomData was finished executing... don't we have this problem in other places?